app/models/service_template.rb

```diff
@@ -416,4 +473,8 @@ def resource_actions_info
     end
     config_info
   end
+
+  def resource_action_attrs
```

Comment:
We need a better name here, this name is confusing/misleading.  Also, I do not see why this is not just a constant instead of a method.

Maybe: `RESOURCE_ACTION_UPDATE_ATTRS`  Open to other suggestions.